README.md

```diff
@@ -78,11 +78,10 @@ export LIBVA_DRIVER_NAME=iHD
 * BDW (Broadwell)
 * SKL (Skylake)
 * BXT (Broxton) / APL (Apollo Lake)
-* KBL (Kaby Lake)
-* CFL (Coffee Lake)
-* WHL (Whiskey Lake)
-* CML (Comet Lake)
+* KBL+ (KBL/Kaby Lake; CFL/Coffe Lake; WHL/Whiskey Lake; CML/Comet Lake; AML/Amber Lake)
```

Comment:
Can you, please, drop +? Someone can read that as everything above KBL gen9 is supported. If you wanted to have abbreviation which will mean a family, then maybe use KBL* or KBLx?